webpack/assets/javascripts/react_app/components/hosts/storage/vmware/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import { Button } from 'react-bootstrap';
 import { connect } from 'react-redux';
-import { omit } from 'lodash';
 
 import Controller from './controller/';
 import * as VmWareActions from '../../../../redux/actions/hosts/storage/vmware';
 import { MaxDisksPerController } from './StorageContainer.consts';
 import './StorageContainer.scss';
 
+const filterKeyFromVolume = volume => Object.entries(volume)
+  .filter(([key]) => key !== 'key')
+  .reduce((vol, [key, val]) => Object.assign(vol, { [key]: val }), {});
```

Comment:
I think that can be simplified to
```js
(volume) => {
  const {key, volumeWithoutKey} = volume;
  return volumeWithoutKey;
}
```
which is a bit more readable to me. What do you think?

Reply:
Or maybe even:
```js
({key, ...volumeWithoutKey}) => volumeWithoutKey;
```